build.gradle

```diff
@@ -2,9 +2,6 @@ buildscript {
     repositories {
         mavenCentral()
         jcenter()
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
     }
```

Comment:
Technically speaking, Gradle 5.6.4 doesn't need a `buildscript` block to configure the plugins any more. This can be done (better!) in `settings.gradle` via a `pluginManagement` block.
```
pluginManagement {
    repositories {
        // You don't need a repositories block at all if you only download plugins from here.
        gradlePluginPortal()
    }

    plugins {
        id 'com.github.johnrengelman.shadow' version shadow_plugin_version
        id "com.dorongold.task-tree" version "1.3.1"
        ...
    }
}
```
And you can define `shadow_plugin_version` in `gradle.properies`.